src/core/document.js

```diff
@@ -821,103 +821,95 @@ class PDFDocument {
     });
   }
 
-  get xfaDatasets() {
+  get _xfaStreams() {
     const acroForm = this.catalog.acroForm;
     if (!acroForm) {
-      return shadow(this, "xfaDatasets", null);
+      return null;
     }
 
     const xfa = acroForm.get("XFA");
+    const entries = {
+      "xdp:xdp": null,
+      template: null,
+      datasets: null,
+      config: null,
+      connectionSet: null,
+      localeSet: null,
+      stylesheet: null,
+      "/xdp:xdp": null,
+    };
```

Comment:
Is there a particular reason for using `null` here rather than empty strings as before, since then you could skip the explicit assign of empty strings in `get xfaData()` below?